packages/hardhat-plugin-ethers/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "@nomicfoundation/hardhat-ignition-ethers",
+  "version": "0.11.2",
+  "license": "MIT",
+  "author": "Nomic Foundation",
+  "homepage": "https://hardhat.org",
+  "description": "The Ethers extension to Hardhat Ignition. Hardhat Ignition is a declarative system for deploying smart contracts on Ethereum. It enables you to define smart contract instances you want to deploy, and any operation you want to run on them. By taking over the deployment and execution, Hardhat Ignition lets you focus on your project instead of getting caught up in the deployment details.",
+  "repository": "github:NomicFoundation/hardhat-ignition",
+  "keywords": [
+    "ethereum",
+    "smart-contracts",
+    "hardhat",
+    "blockchain",
+    "dapps",
+    "tooling",
+    "solidity",
+    "deployment",
+    "ethers"
+  ],
+  "main": "dist/src/index.js",
+  "types": "dist/src/index.d.ts",
+  "files": [
+    "dist/src/",
+    "src/",
+    "LICENSE",
+    "README.md",
+    "CHANGELOG.md"
+  ],
+  "exports": {
+    ".": "./dist/src/index.js",
+    "./modules": "./dist/src/modules.js"
+  },
+  "typesVersions": {
+    "*": {
+      "modules": [
+        "./dist/src/modules.d.ts"
+      ]
+    }
+  },
+  "scripts": {
+    "build": "tsc --build",
+    "lint": "npm run prettier -- --check && npm run eslint",
+    "lint:fix": "npm run prettier -- --write && npm run eslint -- --fix",
+    "eslint": "eslint \"src/**/*.{ts,tsx}\" \"test/**/*.{ts,tsx}\"",
+    "prettier": "prettier \"**/*.{js,ts,md,json}\"",
+    "test": "mocha --recursive \"test/**/*.ts\"",
+    "test:debug": "DEBUG='hardhat-ignition:*,ignition:*' npm run test",
+    "test:coverage": "nyc mocha \"test/**/*.ts\"",
+    "test:build": "tsc --project ./test/",
+    "clean": "rimraf .nyc_output coverage dist tsconfig.tsbuildinfo ./test/fixture-projects/**/deployments ./test/fixture-projects/**/artifacts",
+    "prepack": "npm run build"
+  },
+  "devDependencies": {
+    "@nomicfoundation/hardhat-ethers": "^3.0.4",
+    "@types/chai": "^4.2.22",
+    "@types/chai-as-promised": "^7.1.4",
+    "@types/sinon": "^10.0.13"
+  },
+  "peerDependencies": {
+    "@nomicfoundation/hardhat-ethers": "^3.0.4",
+    "hardhat": "^2.18.0"
+  },
+  "dependencies": {
+    "@nomicfoundation/ignition-core": "^0.11.2",
+    "@nomicfoundation/hardhat-ignition": "^0.11.2",
+    "ethers": "^6.7.0"
```

Comment:
peer dependencies

Reply:
ignition-core is also part of the public API, the modules, so it should be peer too